sparse.c

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) 2016, Bjorn Andersson <bjorn@kryo.se>
```

Comment:
Is this copyright correct ? Surely it should be your name ?

Reply:
just copy-paste, I don't know what it's supposed to be

Reply:
@ndechesne the current file was made by me, so I put my name in it